src/JustSaying/Messaging/Channels/SubscriptionGroups/ISubscriptionGroupSettings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace JustSaying.Messaging.Channels.SubscriptionGroups
+{
+    public interface ISubscriptionGroupSettings
```

Comment:
`///` for the interface please.

Reply:
You can try and flush some of these locally by adding `<GenerateDocumentationFile>true</GenerateDocumentationFile>` to the `.csproj`, which will highlight them as compiler warnings (as well as invalid syntax). It's not turned on by default because otherwise there's hundreds of warnings from all the existing code that doesn't have docs.